pandas/core/frame.py

```diff
@@ -241,17 +241,52 @@ class DataFrame(NDFrame):
         Column labels to use for resulting frame. Will default to
         np.arange(n) if no column labels are provided
     dtype : dtype, default None
-        Data type to force, otherwise infer
+        Data type to force. Only a single dtype is allowed. If None, infer
```

Comment:
https://github.com/pandas-dev/pandas/pull/13375 implemented his for astype
hasn't been implemented for construction